nf/main.nf

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env nextflow
+
+include { SENTIEON_DNASCOPE } from './modules/local/sentieon/dnascope/main'
+include { SENTIEON_DNASCOPE as SENTIEON_DNASCOPE_DIPLOID } from './modules/local/sentieon/dnascope/main'
+include { GATK_INTERVALLISTOOLS } from './modules/local/gatk/intervallisttools/main'
+include { PICARD_MERGEVCFS } from './modules/local/picard/mergevcfs/main'
+include { SENTIEON_GVCFTYPER } from './modules/local/sentieon/gvcftyper/main.nf'
+
+workflow {
+    main:
+    alignment = params.alignment ? Channel.fromPath(params.alignment) : Channel.value([])
+    align_index = params.align_index ? Channel.fromPath(params.align_index) : Channel.value([])
+    reference = Channel.fromPath(params.reference).first()
+    reference_index = Channel.fromPath(params.reference_index).first()
+    wgs_intervals = Channel.fromPath(params.wgs_intervals).first()
+    non_diploid_intervals = Channel.fromPath(params.non_diploid_intervals).first()
+    non_diploid_ploidy = params.non_diploid_ploidy
+    dbsnp = params.dbsnp ? Channel.fromPath(params.dbsnp) : Channel.value([])
+    dbsnp_index = params.dbsnp_index ? Channel.fromPath(params.dbsnp_index) : Channel.value([])
+    indexed_alignment = alignment.combine(align_index)
+    reference_fai = reference.combine(reference_index)
```

Comment:
Kind of confusing. `reference_fai` leads me to believe its a `fasta.fai` when in fact this is the reference plus its fai.